app/controllers/verify/usps_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Verify
+  class UspsController < ApplicationController
+    include IdvStepConcern
+
+    before_action :confirm_step_needed
+
+    helper_method :applicant
```

Comment:
why not make this an instance variable instead? In general I find it less mysterious to have everything in the view be an ivar or a method on an ivar.